extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -186,6 +203,12 @@ export class AmpLiveList extends AMP.BaseElement {
 
     this.manager_ = liveListManagerForDoc(this.getAmpDoc());
 
+    this.listBuilderId_ = this.element.hasAttribute('built-by') ?
```

Comment:
optional: It might be more performant to use a property instead of an attribute.
required: If we stick with an attribute, this should be internally-named (`i-amphtml-...`), so that publishers don't depend on it.

Reply:
Done.